src/xrGame/DiscordRPC.cpp

```diff
@@ -0,0 +1,35 @@
+#include "stdafx.h"
+#include "DiscordRPC.h"
+#include <ctime>
+
+DiscordRPC* g_DiscordRPC;
+
+void DiscordRPC::Init()
+{
+    DiscordEventHandlers Handle;
+    memset(&Handle, 0, sizeof(Handle));
```

Comment:
1. X-Ray uses ZeroMemory function for zeroing memory instead of memset and we use it too.
2. We don't even need manual memory zeroing. Just uses braces initializer;
3. First letter for local variable name should be lowercase. (we use `camelCase` for local variables. `snake_case` is allowed sometimes too)

```suggestion
    DiscordEventHandlers handle{};
```